Add installation instructions for conda (#468)
peak/s5cmd: README.md

## Installation

### Official Releases

#### Binaries

The [Releases](https://github.com/peak/s5cmd/releases) page provides pre-built
binaries for Linux, macOS and Windows.

#### Homebrew

For macOS, a [homebrew](https://brew.sh) tap is provided:

    brew install peak/tap/s5cmd

### Unofficial Releases (by Community)
> **Warning**  

> > These releases are maintained by the community. They might not be up to date or same with the original releases.

Reviewer: ```suggestion
> These releases are maintained by the community. They might be out of date compared to the official releases.
```
